simulator/examples/image-bmp.rs

```diff
@@ -10,11 +12,14 @@ use tinybmp::Bmp;
 fn main() -> Result<(), core::convert::Infallible> {
     let mut display: SimulatorDisplay<Rgb565> = SimulatorDisplay::new(Size::new(128, 128));
 
+    // Load the BMP image
     let bmp = Bmp::from_slice(include_bytes!("./assets/rust-pride.bmp")).unwrap();
 
-    let image: Image<Bmp, Rgb565> = Image::new(&bmp, Point::zero());
+    // Create a new embedded-graphics Image, wrapping the BMP which provides pixel data. The top
```

Comment:
The comment could mention that it's important to annotate the `Image` with the color format used in the image, because otherwise the compiler might infer a wrong color type.